.moban.dt/core-test-requirements.txt.jj2

```diff
@@ -0,0 +1,2 @@
+{% include 'test-requirements.txt.jj2' %}
+ruamel.yaml~=0.15.42
```

Comment:
no EOL at EOF

and if you look in ` .moban.dt/` you will see that the filename prefix `core-` is inappropriate.